test/extensions/filters/http/composite/composite_filter_integration_test.cc

```diff
@@ -276,6 +313,9 @@ TEST_P(CompositeFilterIntegrationTest, TestBasic) {
 // Verifies that if we don't match the match action the request is proxied as normal, while if the
 // match action is hit we apply the specified dynamic filter to the stream.
 TEST_P(CompositeFilterIntegrationTest, TestBasicDynamicFilter) {
+  if (!downstream_filter_) {
```

Comment:
Only support static composite filter in upstream now.  Will remove this check in the follow up PR when support dynamic composite filter in upstream.